system-variables.md

```diff
@@ -4150,6 +4150,10 @@ SHOW WARNINGS;
 
 ### tidb_replica_read <span class="version-mark">New in v4.0</span>
 
+>**Note:**
```

Comment:
It's nice to have this note as cross reference. 

Unfortunately, all the rest variables and statements did not have this. We should add them all or none.

Reply:
Agreed.

Reply:
If you search `TiDB Serverless` on https://docs.pingcap.com/tidb/dev/system-variables, you'll find there are more than 60 instances that indicate the TiDB Serverless support info. 

For example:
![image](https://github.com/pingcap/docs/assets/79440533/e57b985a-f476-429a-8beb-a790267908b3)

Therefore, it looks OK to add the support info for this variable too?